keyboards/dz60/keymaps/billiams/keymap.c

```diff
@@ -0,0 +1,50 @@
+#include QMK_KEYBOARD_H
+
+#define ______ KC_TRNS
```

Comment:
QMK defines an alias for this keycode automatically.

https://github.com/qmk/qmk_firmware/blob/d3268280ef7077ec1c800d30b8e98f1976192a14/quantum/quantum_keycodes.h#L29-L31

Please delete Line 3, and replace all instances of  
`______` (six underscores) with  
`_______` (seven underscores).

Reply:
@noroadsleft okay done